lib/kafka_rest_client.js

```diff
@@ -72,7 +74,8 @@ function KafkaRestClient(options, callback) {
     self.discoverTopics(function discoverTopicsCallback(err, isDone) {
         if (err || !isDone) {
             self.enable = false;
-            callback(new Error('Failed to discover topics mapping'));
+            self.scheduleReconnect(0);
```

Comment:
Add a comment here explaining that we won't return an error (since there are retries)